lib/final-api/endpoint/ddtf.rb

```diff
@@ -64,7 +64,7 @@ def registered(app)
 
             halt 400, enqueue_data.errors.to_json unless enqueue_data.valid?
 
-            config = DdtfHelpers.build_config(payload, tsd)
+            config = DdtfHelpers.build_config(payload, JSON.parse(enqueue_data.enqueue_data['TsdContent']))
```

Comment:
I do not like this change:)

`tsd` variable is `enqueue_data.load_tsd` so this change introduce two different ways how to get tsd form payload. Can you use tsd_utils #lload_tsd method and modify it<